Server/package.json

```diff
@@ -12,7 +12,10 @@
   "dependencies": {
     "body-parser": "^1.19.0",
     "express": "^4.17.1",
-    "mongoose": "^5.6.0"
+    "mongoose": "^5.6.0",
+    "moment": "^2.24.0"
   },
-  "devDependencies": {}
+  "devDependencies": {
```

Comment:
Refer https://github.com/shashank-compro/time-management-system/pull/3/files#r325643016

Reply:
Removed for now, will take a pull from latest